src/example-types/dateRangeFacet.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
         range: {
           date_range: {
             field,
-            format: format || "yyyy-MM-dd'T'HH:mm:ss.SSSZ",
```

Comment:
on ES7 this breaks and requires a lowercase `z`, but on ES6 this needs to be uppercase and breaks with uppercase `Z`. Thanks to @lusid for the idea to just switch to [`dateOptionalTime` which is basically a standard ISO parser](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html)